files/chef-marketplace-cookbooks/chef-marketplace/templates/default/reckoner.rb.erb

```diff
@@ -1,16 +1,17 @@
 updater.driver = '<%= node['chef-marketplace']['platform'] == 'aws' ? 'ec2' : node['chef-marketplace']['platform'] %>'
 checker.driver = '<%= node['chef-marketplace']['role'] == 'aio' ? 'chef_server' : 'compliance' %>'
-license.count = <%= node['chef-marketplace']['license_count'] %>
+license.count = <%= count = node['chef-marketplace']['license_count']; count.is_a?(String) ? "'#{count}'" : count %>
```

Comment:
Summarizing our conversation, I think it would be better to have a `license.type` of either "fixed" or "fcp" (or whatever) and then `license.count` is only output if the type is "fixed"